v2/src/lib.rs

```diff
@@ -275,13 +286,13 @@ impl Volume for InfluxDbVolume {
         };
 
         //check if db exists, if it doesn't create one if user has set createdb=true in config
-        match async_std::task::block_on(async { is_db_existing(&client, &db).await }) {
+        match TOKIO_RUNTIME.block_on(async { is_db_existing(&client, &db).await }) {
             Ok(res) => {
                 if !res && createdb {
                     // try to create db using user credentials
-                    match async_std::task::block_on(async {
-                        create_db(&self.admin_client, &creds.org_id, &db).await
-                    }) {
+                    match TOKIO_RUNTIME
+                        .block_on(async { create_db(&self.admin_client, &creds.org_id, &db).await })
```

Comment:
`block_on` should as well try to get `current` runtime in the same way as done for `spawn`.